src/index.test.ts

```diff
@@ -25,6 +25,7 @@ import {
   useNavigationModalDismiss,
   useNavigationScreenPop,
   useNavigationBottomTabSelect,
+  useNavigationBottomTabPress,
```

Comment:
I think you should upgrade RNN version and think about introducing a breaking change here as we did in the previous [v5.0.0](https://github.com/underscopeio/react-native-navigation-hooks/releases/tag/v5.0.0)